lib/workers/pr/body/controls.ts

```diff
@@ -1,3 +1,9 @@
-export function getControls(): string {
-  return `\n\n---\n\n - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box\n\n`;
+import { isBranchModified } from '../../../util/git';
+import { BranchConfig } from '../../types';
+
+export async function getControls(config: BranchConfig): Promise<string> {
+  const warning = (await isBranchModified(config.branchName))
+    ? ' :warning: **Warning**: custom changes will be lost.'
```

Comment:
should this use emojify?

Reply:
Based on other similar code, I think so

Reply:
Cool. I thought so too after so it’s already done. Forgot to resolve this